gpflow/kernels/mo_kernels.py

```diff
@@ -76,10 +85,11 @@ class SharedIndependent(MultioutputKernel):
     Use `gpflow.kernels` instead for more efficient code.
     """
     def __init__(self, kernel: Kernel, output_dimensionality: int):
-        super().__init__()
+        super().__init__(output_dim=output_dimensionality)
```

Comment:
```suggestion
        super().__init__(output_dim=output_dim)
```

Reply:
https://github.com/GPflow/GPflow/pull/1167/files/cd98ac251922e80a1c374e1213434445347aba53#r353256360

Reply:
same as my comment though i agree its not _truly_ cooperative w `super()`